x-pack/legacy/plugins/ml/public/data_frame_analytics/pages/analytics_management/components/create_analytics_advanced_editor/create_analytics_advanced_editor.tsx

```diff
@@ -158,27 +160,32 @@ export const CreateAnalyticsAdvancedEditor: FC<CreateAnalyticsFormProps> = ({ ac
               <EuiSpacer />
             </Fragment>
           ))}
-          <EuiFormRow
-            isInvalid={createIndexPattern && destinationIndexPatternTitleExists}
-            error={
-              createIndexPattern &&
-              destinationIndexPatternTitleExists && [
-                i18n.translate('xpack.ml.dataframe.analytics.create.indexPatternTitleError', {
-                  defaultMessage: 'An index pattern with this title already exists.',
-                }),
-              ]
-            }
-          >
-            <EuiSwitch
-              disabled={isJobCreated}
-              name="mlDataFrameAnalyticsCreateIndexPattern"
-              label={i18n.translate('xpack.ml.dataframe.analytics.create.createIndexPatternLabel', {
-                defaultMessage: 'Create index pattern',
-              })}
-              checked={createIndexPattern === true}
-              onChange={() => setFormState({ createIndexPattern: !createIndexPattern })}
-            />
-          </EuiFormRow>
+          {jobType === JOB_TYPES.OUTLIER_DETECTION && (
+            <EuiFormRow
+              isInvalid={createIndexPattern && destinationIndexPatternTitleExists}
+              error={
+                createIndexPattern &&
+                destinationIndexPatternTitleExists && [
+                  i18n.translate('xpack.ml.dataframe.analytics.create.indexPatternTitleError', {
```

Comment:
Maybe give a more specific label here, like `indexPatternAlreadyExistsError`, in case we need to show any more index pattern related errors.

Reply:
Done in 6de9d0d162db3f4225bd32b347bdd502f4d51719